src/magic_spell_effect.cpp

```diff
@@ -425,6 +425,12 @@ static void damage_targets( const spell &sp, Creature &caster,
         }
         sp.make_sound( target );
         sp.create_field( target );
+        if( sp.has_flag( spell_flag::IGNITE_FLAMMABLE ) ) {
+            const float fire_damage = sp.get_damage_instance().type_damage( DT_HEAT );
```

Comment:
please remove this check for heat damage type, even though it may not make perfect sense, the flag is already there to prevent it. no need to add the extra check, and magic is magic.

Reply:
ok, done.